x-pack/plugins/code/public/components/symbol_tree/props.ts

```diff
@@ -4,251 +4,90 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { SymbolKind } from 'vscode-languageserver-types/lib/esm/main';
+import { SymbolKind } from 'vscode-languageserver-types/lib/umd/main';
```

Comment:
you should just need to do import { SymbolKind } from 'vscode-languageserver-types'